src/modules/Root/Root.jsx

```diff
@@ -22,15 +23,17 @@ class Root extends Component {
           <Header>
             <NavLink exact to="/" className="j-nav__item" activeClassName="j-nav__item--active">API Definitions</NavLink>
             <NavLink to="/new" className="j-nav__item" activeClassName="j-nav__item--active">New API</NavLink>
+            <NavLink to="/log-in" className="j-nav__item" activeClassName="j-nav__item--active">Login Page</NavLink>
```

Comment:
 yes!!!!! My bad!  Just start work on this before **bem-cn** was implemented and than just forget to use it in this PR.